plugins/org.yakindu.sct.doc.user/src/user-guide/generating_code_headless.textile

```diff
@@ -203,13 +201,3 @@ bc.
 plugins/org.eclipse.equinox.launcher.gtk.linux.x86_64_1.1.400.v20160518-1444
 
 Replace the path in the _headless.ini_ file with the path from _eclipse.ini_. After saving _headless.ini_, _scc_ should work.
-
-h4(#hdls_troubleshooting_could_not_find_project_file). [ERROR] Cannot find any .project files
-
-Upon execution _scc_ searches for _.project_ files which are used to properly interpret generation target locations as specified in the _sgen_ files. These _.project_ files are hidden files that contain some meta information about your projects in your YAKINDU Statechart Tools workspace. If _scc_ does not find these files you might have copied your models to another folder and missed to also copy these files.
-
-h4(#hdls_troubleshooting_could_not_resolve_reference_to_eobject). [ERROR] Couldn't resolve reference to EObject
-
-This error usually denotes a configuration problem when using _scc_ in combination with the deep C integration feature. For example, if you are referring to imported C elements in your statechart models, and the corresponding header files can no more be resolved on your CI build server, the linker is unable to find the referenced C elements and produces this error message. To overcome this problem, you might want to check if your settings in _C/C++ General -> Path and Symbols_ do not contain any paths that do not exist on your build environment. Also make sure that _scc_ can access these settings which are stored in the _.cproject_ file.
-
-==</div>==
```

Comment:
why are these parts removed?

Reply:
Hm, seems to be collateral damage from the merge process. I reinstated it and checked the rest, too. Should be okay now.